token-swap/program/src/instruction.rs

```diff
@@ -83,6 +88,7 @@ pub enum SwapInstruction {
     ///   6. `[writable]` token_a user Account to credit.
     ///   7. `[writable]` token_b user Account to credit.
     ///   8. '[]` Token program id
+    ///   9. `[writable]` Fee account, to receive withdrawal fees
```

Comment:
This parameter is last per Bartosz's request

Reply:
Any particular reason to not have `Token program id` last, as in the other instructions?
Edit: noticed that token program id is actually not last in swap, just in swap docs. I still rather like program id last.